src/runtime/runtime.cs

```diff
@@ -130,10 +141,21 @@ internal static void Initialize(bool initSigs = false, ShutdownMode mode = Shutd
                 {
                     PyGILState_Ensure();
                 }
+
+                BorrowedReference pyRun = PySys_GetObject("__pynet_run__");
```

Comment:
1. Maybe this string should be interned?
2. We use `pythonnet` everywhere else, makes sense to use that here, too
3. Is it possible to put this in `clr` instead?

Reply:
1. There's no point of interning a string, that is not used often.
2. Will change
3. > Is it possible to put this in `clr` instead?

I think not, because the initialization of `CLRModule` involves `PyObject` instances, so needs run number.